CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,129 @@
+
+* Responsibilities:
+    * Continues to contribute regularly
+    * Work to accomplish the tasks they volunteer to do within the project
+
+* Qualifications:
+    * Must have successful contributions to the project, including at least one of the following:
+        * Written and/or reviewed PRs,
+        * Reviewed PRs,
+        * Made contributions that resolved Issues,
+        * Or some equivalent combination or contribution
+    * Must be actively contributing to at least one project area, where the privileges of Organization Member will be beneficial
+
+* Privileges:
+    * Can trigger CI
+    * Can leave reviews on PRs
+    * Can recommend other contributors to become Cilium Github Org Members
+    * Can nominate themselves to other roles within Cilium
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Open a PR in the member management file
+2. Receive two "LGTM" comments from any Org Members, and approval by a committer who merges the PR
+
+### Reviewer
+Description: A Reviewer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Reviewers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+Reviewers are responsible for a "specific area." This can be a specific code directory, chapter of the docs, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+Reviewers have all the rights and responsibilities of an Organization Member, plus:
+* Responsibilities:
+    * Following the [reviewing guide](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#pull-requests-review-process-for-committers) 
+    * Regularly reviewing Pull Requests against their specific areas of responsibility, including Pull Requests that are assigned to them
+    * Helping other contributors become reviewers
+    * Review CFPs and provide feedback
+
+* Requirements:
+    * Is an Organization Member
+    * Has a track record of constructive and valuable reviews
+    * Has demonstrated an in-depth knowledge of the specific area, for example by resolving test failures or through contributions
+    * Commits to being responsible for that specific area
+    * Is supportive of new and occasional contributors and helps get useful PRs in shape to commit
+* Privileges:
+    * Has GitHub rights to approve pull requests in specific directories
+    * Can recommend and review other Org Members to become Reviewers
+
+The process for an Organization Member to become a Reviewer is as follows:
+
+1. Open a PR in the member management file for the correct team
```

Comment:
Perhaps we should have two categories here, one that makes it easy to apply permissions for all reviewers and one for a particular code area that the org member intends to participate in?

```suggestion
1. Open a PR in the member management file for at least two teams: "Reviewers" and for a specific review area
```